Source/Lib/Codec/EbTime.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright(c) 2019 Intel Corporation
+ * SPDX - License - Identifier: BSD - 2 - Clause - Patent
+ */
+
+#ifndef EbTime_h
+#define EbTime_h
+
+#ifdef _WIN32
+#include <windows.h>
+#include <sys/timeb.h>
+#elif !defined(__USE_POSIX199309)
+#define __USE_POSIX199309
+#endif
+
+#include <stdint.h>
+#include <time.h>
+
+#if !defined(CLOCK_MONOTONIC) && !defined(_WIN32)
+#include <sys/time.h>
+#endif
+
+static inline void svt_vp9_sleep(const int milliseconds) {
```

Comment:
To keep coding convention in SVT, we separate declaration and definition in .h and .c file.
Also use either `static` or `inline`, both of them together are not necessary.

Reply:
Done